app/views/govuk_publishing_components/components/docs/contents_list.yml

```diff
@@ -209,6 +209,22 @@ examples:
           text: Fourth page title link
         - href: "#fifth-thing"
           text: Fifth page title link
+  with_alternate_title:
```

Comment:
It's hard naming things but maybe `with_custom_title` would be clearer?

Reply:
As above, I could change this to just `custom-title`? I used the `with-` prefix to match other existing options, but there isn't much consistency in how they're named.

Reply:
Sure.

Reply:
Rebasing went a bit awry, but after a bit of multi-branch cherry-picking, this is now changed to 'with-custom-title`.